src/Morphir/Visual/ViewValue.elm

```diff
@@ -96,6 +102,15 @@ viewValueByValueType ctx argumentValues typedValue =
         in
         ViewBoolOperatorTree.view (viewValueByLanguageFeature ctx argumentValues) boolOperatorTree
 
+    else if Basics.isNumber valueType then
+        let
+            arithmeticOperatorTree : ArithmeticOperatorTree
+            arithmeticOperatorTree =
+                ArithmeticOperatorTree.fromArithmeticTypedValue typedValue
+        in
+        ViewArithmetic.view (viewValueByLanguageFeature ctx argumentValues) arithmeticOperatorTree
+        -- Element.column [] [ ViewArithmetic.view (viewValueByLanguageFeature ctx argumentValues) arithmeticOperatorTree, Element.html (div [ Html.Attributes.style "display" "none", Html.Attributes.style "padding" "10", Html.Attributes.style "background-color" "rgb (50 50 50)", Html.Attributes.style "font-size" "16", Html.Attributes.style "color" "rgb(255 78 185 255)" ] [ outerView [] (parseArithmeticOperatorTree arithmeticOperatorTree) ]) ]
```

Comment:
Can you remove these comments.

Reply:
Moved comment to AbstractTreeVisualizer.elm,
can be added back here to get visual of Arithmetic AST (debugging)